packages/components/src/components/grid/grid-item/grid-item-styles.ts

```diff
@@ -1,21 +1,23 @@
-import type { GridGutter } from '../grid/grid-utils';
 import type { GridItemOffset, GridItemSize } from './grid-item-utils';
-import { buildResponsiveStyles, getCss, mergeDeep } from '../../../utils';
-import { addImportantToEachRule, pxToRemWithUnit } from '../../../styles';
 import type { BreakpointCustomizable } from '../../../types';
+import { buildResponsiveStyles, getCss, mergeDeep } from '../../../utils';
+import { addImportantToEachRule } from '../../../styles';
+import { gridGap } from '@porsche-design-system/utilities-v2';
 
+const gutter = `calc(${gridGap} / 2)`;
```

Comment:
Could be shared instead of redefined.

Reply:
I don't think it will save a lot since the other gutter needs to be negative.